x-pack/plugins/apm/server/lib/connections/get_connection_stats/get_stats.ts

```diff
@@ -55,7 +56,12 @@ export const getStats = async ({
 
   const response = await apmEventClient.search('get_connection_stats', {
     apm: {
-      events: [ProcessorEvent.metric],
+      sources: [
+        {
+          documentType: ApmDocumentType.ServiceDestinationMetric,
+          rollupInterval: RollupInterval.OneMinute,
```

Comment:
Why is the `rollupInterval` hard-coded in this case?

Reply:
In this case I didn't use `usePreferredDataSourceAndBucketSize` because `getConnectionStats` it's used in several places and I did a lot of changes in different files, not remember where, it gave me some problems, so I decided to just modify `get_stats.ts`. But I can take a look again in follow-ups to try use the mentioned function.